storage/errors.go

```diff
@@ -0,0 +1,369 @@
+// Copyright 2020 Coinbase, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package storage
+
+import (
+	"errors"
+
+	utils "github.com/coinbase/rosetta-sdk-go/errors"
+)
+
+// Named error types for Storage errors
+var (
+	////////////////////
+	/* BADGER STORAGE */
+	////////////////////
+
+	ErrDatabaseOpenFailed         = errors.New("could not open database")
```

Comment:
nit: to be consistent with other error messages. Maybe it's better to update to the following
```suggestion
	ErrDatabaseOpenFailed         = errors.New("unable to open database")
```